x-pack/plugins/lens/common/expressions/xy_chart/xy_chart.ts

```diff
@@ -174,7 +193,22 @@ export const xyChart: ExpressionFunctionDefinition<
   },
   fn(data: LensMultiTable, args: XYArgs, handlers) {
     if (handlers?.inspectorAdapters?.tables) {
-      logDataTable(handlers.inspectorAdapters.tables, data.tables);
+      args.layers.forEach(({ layerId, accessors, xAccessor, splitAccessor, layerType }) => {
```

Comment:
I just merged main to this branch and typescript will complain about these params, because annotation layer doesn't contain any of these (plus it will break because you try to access nonexisting property). You could simply filter it out from the layers by using `!isAnnotationsLayer(layer)` because it shouldn't be in the inspector anyway, right?